src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,41 +1,36 @@
 package mate.academy.hibernate.relations.model;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@ToString
+@Getter
+@Setter
+@NoArgsConstructor
+@Entity
+@Table(name = "actors")
 public class Actor implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String name;
+    @ManyToOne
+    @JoinColumn(name = "country_id")
     private Country country;
 
-    public Actor() {
-    }
-
     public Actor(String name) {
         this.name = name;
     }
```

Comment:
FYI: if u wanna use lombok u can use `Data` annotation
`ToString` + `Getter` + `Setter` + `Equals` + `HashCode` == `Data`
Also, u can put annotation `NonNull` above filed name and use annotation `RequiredArgsConstructor`